src/Compilers/CSharp/Test/Emit/PDB/PDBTests.cs

```diff
@@ -5767,6 +5767,96 @@ static void M()
 
         #endregion
 
+        #region OutVar
+
+        [Fact]
+        public void SyntaxOffset_OutVarInConstructor()
```

Comment:
This is for completeness. If we allow out var in initializers in future this test will be updated to check that the debug info is right.